app/src/main/assets/applozic-settings.json

```diff
@@ -85,6 +85,7 @@
   "toolbarColor": "",
   "statusBarColor": "",
   "richMessageThemeColor": "",
+  "hideKeyboardForBot": false,
```

Comment:
are we hiding only keyboard while continuing to show the textfield? or we are hiding the whole?

Reply:
Hiding the whole. The keyboard would hide automatically once the text field is hidden

Reply:
![Screenshot_20210304-212004](https://user-images.githubusercontent.com/9147912/109990655-9adc5d00-7d2f-11eb-98e0-b831ae200a65.jpg)